### ISSUE TITLE
Add vignette or tab section for videos or external use cases

### ISSUE THREAD

**Issue by avallecam** [epiverse-trace/finalsize#120]
An idea that could be of interest after having case studies or video recordings of presentations.

Inspired by the "Videos" tab section in the headline of [gganimate package](https://gganimate.com/articles/extra/talks.html) and indexed "scientific use cases" by rOpenSci for the [osmdata package](https://ropensci.org/packages/geospatial/)

**Comment by pratikunterwegs** [comment]
I don't think we have any such videos right now, but if the Epiverse showcase from next week is recorded and uploaded, it could be embedded here.
